### PR TITLE
Addressing feedback from code review (21/09/2018)

### PR DESCRIPTION
**Pull request by Ivanca**
- Creating .in file for pip-compile (to generate requirements.txt)
- Moving pytest.ini settings to setup.cfg (removes pytest.ini)
- Uncomment on Makefile 'clean' : rf -rf *.egg-info
- Moving coverage.rc settings to setup.cfg (removes coverage.rc)
- test_utils no longer used @edx.org domain (uses example.com)
- README.md improved with usage instructions and better install instructions

### CODE REVIEW

**Comment by Ivanca** [comment]
Now tests are passing @felipemontoya , I had to specify that the dependency of a dependency is only for python 2.7:
```python
futures==3.2.0 ; python_version < "3.0"
```

File: requirements.in [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+pytest==3.6.3
+pytest-django==3.3.3
+mock==2.0.0
+factory-boy==2.10.0
+pylint==1.9.3
+futures==3.2.0 ; python_version < "3.0"
+
+# Oauth2: These 2 packages both are needed
+-e git+https://github.com/edx/django-rest-framework-oauth.git@0a43e8525f1e3048efe4bc70c03de308a277197c#egg=djangorestframework-oauth==1.1.1
+django-oauth2-provider==0.2.6.1
```
Comment (felipemontoya) [inline requirements.in]: missing the final empty line

File: setup.cfg [hunk 2]
```diff
@@ -5,3 +5,10 @@ tag = True
 
 [bumpversion:file:eox_core/__init__.py]
 
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = eox_core.settings.test
+
+[coverage:run]
+data_file = .coverage
+omit =
+    venv/*
```
Comment (felipemontoya) [inline setup.cfg]: emtpy line at the end. Why is the linting not failing because of this?

Comment (felipemontoya) [reply setup.cfg]: I see. Because this is not a .py file....

Comment (Ivanca) [reply setup.cfg]: Empty line added.

**Pull request merged**